src/main/java/competition/subsystems/collector/commands/ExtendCollectorCommand.java

```diff
@@ -0,0 +1,24 @@
+package competition.subsystems.collector.commands;
+
+import competition.subsystems.collector.CollectorSubsystem;
+import xbot.common.command.BaseCommand;
+
+import javax.inject.Inject;
+
+public class ExtendCollectorCommand extends BaseCommand {
+    CollectorSubsystem collector;
+    @Inject
+    public ExtendCollectorCommand(CollectorSubsystem collector){
+        this.collector = collector;
+    }
+    @Override
+    public void initialize() {
```

Comment:
We should make this only extend in `initialize` and have an `isFinished` that returns true, since this operations is essentially instantaneous.